zerver/tests/test_events.py

```diff
@@ -2825,7 +2825,7 @@ def test_realm_bots_non_admin(self) -> None:
         self.assertNotIn(api_key, str(result))
 
     # Admin users have access to all bots in the realm_bots field
-    def test_realm_bots_admin(self) -> None:
+    def test_realm_bots_e(self) -> None:
```

Comment:
I was just glancing over this and have a quick question: is this function name a typo?

Reply:
It does look to be; want to add a quick commit fixing this?

Reply:
Opened a small [PR](https://github.com/zulip/zulip/pull/13310) for this.